src/oteltest/viz/__init__.py

```diff
@@ -61,7 +102,7 @@ def __init__(self, trace_dir: str):
         self.trace_dir = Path(trace_dir)
         self.app = Flask(__name__)
         self.app.add_url_rule("/", "index", self.index)
-        self.app.add_url_rule("/trace/<path:filename>", "view_trace", self.view_trace)
+        self.app.add_url_rule("/trace/<path:filename>", "view_telemetry", self.view_telemetry)
```

Comment:
Now that this app handles more than just traces it should probably be migrated to be more signal agnostic. So e.g. "/trace/foo.0.json" perhaps should be changed to "/file/foo.0.json" or something. Also, I'm thinking the name of the class -- currently `TraceApp` -- should probably also be changed to `VizApp` or something signal agnostic.

However, this is all pre-existing and perhaps out of scope for this PR. Can be done separately.